src/flb_filter.c

```diff
@@ -393,6 +395,8 @@ int flb_filter_init_all(struct flb_config *config)
         /* Register filter metrics */
         flb_metrics_add(FLB_METRIC_N_DROPPED, "drop_records", ins->metrics);
         flb_metrics_add(FLB_METRIC_N_ADDED, "add_records", ins->metrics);
+        flb_metrics_add(FLB_METRIC_N_RECORDS, "records", ins->metrics);
```

Comment:
what about records_total && bytes_total ?

Reply:
Works for me. Do you prefer a followup commit or a squashed commit?

Reply:
The `_total` suffix is added by the code in api/v1/metrics.c, so this doesn't need to be added here, see also the example output that shows it.